src/validation.js

```diff
@@ -0,0 +1,30 @@
+// @flow
```

Comment:
!! Please setup IDE to add copyrights for new files:
https://mysteriumnetwork.atlassian.net/wiki/spaces/MYS/pages/238682113/Copyright+headers+setup

I understand CI should be in place to ensure that, but we'll have to setup our IDEs either way :)